pkg/ruler/mapper.go

```diff
@@ -8,10 +8,9 @@ import (
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/prometheus/pkg/rulefmt"
 	"github.com/spf13/afero"
 	"gopkg.in/yaml.v2"
```

Comment:
I think your tests are failing because you are still marshaling using yaml.v2 here